.gitlab/deploy_7/docker.yml

```
@@ -6,319 +6,77 @@
-      If (`$lastExitCode -ne "0") { throw "Previous command returned `$lastExitCode" }
-      inv -e docker.publish-bulk ${SIGNING_ARGS} --platform windows/amd64 --src-template ${SRC_AGENT}:${SRC_TAG}-7-win1909-servercore-ARCH --dst-template ${REPOSITORY}-ARCH:`${VERSION}-win1909-servercore
-      If (`$lastExitCode -ne "0") { throw "Previous command returned `$lastExitCode" }
-      inv -e docker.publish-bulk ${SIGNING_ARGS} --platform windows/amd64 --src-template ${SRC_AGENT}:${SRC_TAG}-7-jmx-win1909-servercore-ARCH --dst-template ${REPOSITORY}-ARCH:`${VERSION}-jmx-win1909-servercore
-      If (`$lastExitCode -ne "0") { throw "Previous command returned `$lastExitCode" }
-      inv -e docker.publish-bulk ${SIGNING_ARGS} --platform windows/amd64 --src-template ${SRC_AGENT}:${SRC_TAG}-7-win2004-servercore-ARCH --dst-template ${REPOSITORY}-ARCH:`${VERSION}-win2004-servercore
-      If (`$lastExitCode -ne "0") { throw "Previous command returned `$lastExitCode" }
-      inv -e docker.publish-bulk ${SIGNING_ARGS} --platform windows/amd64 --src-template ${SRC_AGENT}:${SRC_TAG}-7-jmx-win2004-servercore-ARCH --dst-template ${REPOSITORY}-ARCH:`${VERSION}-jmx-win2004-servercore
-      If (`$lastExitCode -ne "0") { throw "Previous command returned `$lastExitCode" }
-      "@ | Add-Content ci-scripts/publish.ps1
-    - cat ci-scripts/publish.ps1
-    - docker run --rm -w C:\mnt -e AWS_NETWORKING=true -e SIGN_WINDOWS=true -v "$(Get-Location):C:\mnt" -v \\.\pipe\docker_engine:\\.\pipe\docker_engine 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/windows_${Env:VARIANT}_x64:${Env:DATADOG_AGENT_WINBUILDIMAGES} powershell -C C:\mnt\ci-scripts\publish.ps1
-    - If ($lastExitCode -ne "0") { throw "Previous command returned $lastExitCode" }
-
-deploy_docker_hub_windows-a7:
-  extends:
-    - .docker_tag_windows_job_definition
-    - .deploy_docker_windows-a7
   rules:
     !reference [.on_deploy_a7_manual]
-  variables:
-    REPOSITORY: datadog/agent
-    SIGNING_ARGS: --signed-push
-
-deploy_google_container_registry_windows-a7:
-  extends:
-    - .google_container_registry_tag_windows_job_definition
-    - .deploy_docker_windows-a7
-  rules:
-    !reference [.on_deploy_a7_manual]
-  variables:
-    REPOSITORY: gcr.io/datadoghq/agent
-    SIGNING_ARGS: ""
-
-.deploy_docker-dogstatsd:
+  trigger:
+    project: DataDog/public-images
+    branch: main
+    strategy: depend
+  parallel:
+    matrix:
+      - IMG_SOURCES: ${SRC_AGENT}:v$CI_PIPELINE_ID-${CI_COMMIT_SHORT_SHA}-7-amd64,${SRC_AGENT}:v$CI_PIPELINE_ID-${CI_COMMIT_SHORT_SHA}-7-arm64,${SRC_AGENT}:v$CI_PIPELINE_ID-${CI_COMMIT_SHORT_SHA}-7-win1809-amd64,${SRC_AGENT}:v$CI_PIPELINE_ID-${CI_COMMIT_SHORT_SHA}-7-win1909-amd64,${SRC_AGENT}:v$CI_PIPELINE_ID-${CI_COMMIT_SHORT_SHA}-7-win2009-amd64
```

We don't have `2009` images, only `2004` ones:
```suggestion
      - IMG_SOURCES: ${SRC_AGENT}:v$CI_PIPELINE_ID-${CI_COMMIT_SHORT_SHA}-7-amd64,${SRC_AGENT}:v$CI_PIPELINE_ID-${CI_COMMIT_SHORT_SHA}-7-arm64,${SRC_AGENT}:v$CI_PIPELINE_ID-${CI_COMMIT_SHORT_SHA}-7-win1809-amd64,${SRC_AGENT}:v$CI_PIPELINE_ID-${CI_COMMIT_SHORT_SHA}-7-win1909-amd64,${SRC_AGENT}:v$CI_PIPELINE_ID-${CI_COMMIT_SHORT_SHA}-7-win2004-amd64
```